Dockerfile.arm64

```diff
@@ -0,0 +1,61 @@
+FROM ubuntu:16.04
+
+ARG S6_OVERLAY_VERSION=v1.22.1.0
+ARG S6_OVERLAY_ARCH=aarch64
+ARG PLEX_BUILD=linux-aarch64
+ARG PLEX_DISTRO=debian
+ARG DEBIAN_FRONTEND="noninteractive"
+ENV TERM="xterm" LANG="C.UTF-8" LC_ALL="C.UTF-8"
+
+ENTRYPOINT ["/init"]
+
+RUN \
+# Update and get dependencies
+    apt-get update && \
+    apt-get install -y \
+      tzdata \
+      curl \
+      xmlstarlet \
+      uuid-runtime \
+      unrar \
+      udev \
+    && \
+
+# Fetch and extract S6 overlay
+    curl -J -L -o /tmp/s6-overlay-${S6_OVERLAY_ARCH}.tar.gz https://github.com/just-containers/s6-overlay/releases/download/${S6_OVERLAY_VERSION}/s6-overlay-${S6_OVERLAY_ARCH}.tar.gz && \
+    tar xzf /tmp/s6-overlay-${S6_OVERLAY_ARCH}.tar.gz -C / && \
+
+# Add user
+    useradd -U -d /config -s /bin/false plex && \
+    usermod -G users plex && \
+
+# Setup directories
+    mkdir -p \
+      /config \
+      /transcode \
+      /data \
+    && \
+
+# Cleanup
+    apt-get -y autoremove && \
+    apt-get -y clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    rm -rf /tmp/* && \
+    rm -rf /var/tmp/*
+
+EXPOSE 32400/tcp 3005/tcp 8324/tcp 32469/tcp 1900/udp 32410/udp 32412/udp 32413/udp 32414/udp
+VOLUME /config /transcode
+
+ENV CHANGE_CONFIG_DIR_OWNERSHIP="true" \
+    HOME="/config"
+
+ARG TAG=public
+ARG URL=
+
+COPY root/ /
+
+RUN \
+# Save version and install
+    /installBinary.sh
+
+HEALTHCHECK --interval=5s --timeout=2s --retries=20 CMD /healthcheck.sh || exit 1
```

Comment:
Can't say I like the idea of having two nearly identical files.  It'd be too easy to update one and forget to update the other.  Though it seems that the way docker is designed, the only solution to fix this is to have a template and then a script which generates the individual dockerfiles.

Reply:
I have taken this idea by browsing different repo about Docker images. And yes, because Docker needs separated Dockerfile for each CPU architecture it's easy to forget one to update. I can also work around this idea of templating the Dockerfile!

Reply:
This was more of a general comment about the way Docker is setup.  Don't see this as a hard requirement for this PR.

Reply:
I get the idea, so what do you think, should we go for a combo template/script or stay with the 2 Dockerfile for that moment?

Reply:
For now likely the 2 Dockerfiles would be better.

Reply:
Okay, that seems legit